data/registry/instrumentation-dotnet-throttlingtroll.yml

```
@@ -0,0 +1,26 @@
+# cSpell:ignore throttlingtroll
+title: ThrottlingTroll
+registryType: instrumentation
+language: dotnet
+license: MIT
+tags:
+  - throttlingtroll
+  - .net
+  - instrumentation
+description:
+  ThrottlingTroll is a rate limiting/throttling/circuit-breaking middleware for
+  ASP.NET Core and Azure Functions. Now instrumented with OTel, so you can
```

```suggestion
  ASP.NET Core and Azure Functions. ThrottlingTroll is instrumented with OTel, so you can
```

Remove the "now", since this description will stay around for a while so I would suggest not making any time-related statements.

---

Thanks @svrnm , that makes sense!